config/locales/en.yml

```diff
@@ -234,3 +235,4 @@ en:
         modern_foreign_languages: "Modern foreign languages"
         physics: "Physics"
         none_of_the_above: "None of the above"
+        
```

Comment:
There is some whitespace here that needs to be removed

Reply:
removed the line